libcontainer/seccomp/seccomp_linux.go

```diff
@@ -170,29 +170,55 @@ func matchCall(filter *libseccomp.ScmpFilter, call *configs.Syscall) error {
 	// Convert the call's action to the libseccomp equivalent
 	callAct, err := getAction(call.Action)
 	if err != nil {
-		return err
+		return fmt.Errorf("action in seccomp profile is invalid: %s", err)
 	}
 
 	// Unconditional match - just add the rule
 	if len(call.Args) == 0 {
 		if err = filter.AddRule(callNum, callAct); err != nil {
-			return err
+			return fmt.Errorf("error adding seccomp filter rule for syscall %s: %s", call.Name, err)
 		}
 	} else {
-		// Conditional match - convert the per-arg rules into library format
+		// If two or more arguments have the same condition,
+		// Revert to old behavior, adding each condition as a separate rule
+		argCounts := make([]uint, 6)
```

Comment:
While I recognise that currently Linux only has 6-argument syscalls, and it's unlikely we'll ever have 7-argument syscalls, can we make this a global `MaxSyscallArguments` somewhere?

Reply:
Should this constant live in runc or the libseccomp bindings? I think the bindings would be the logical place, but it would require revendoring once the change lands.

Reply:
I agree with putting it in the libseccomp bindings, but let's keep in `runc` for now until the change lands.